pysatNASA/instruments/timed_guvi.py

```diff
@@ -186,39 +201,38 @@ def load(fnames, tag='', inst_id=''):
               'fill_val': ('fill', float)}
     meta = pysat.Meta(labels=labels)
 
-    dayset, nightset, aurset = None, None, None
+    # Dimensions for time variables
+    if 'img' in tag:
+        dims = ['nAlongDay', 'nAlongNight', 'nAlongDayAur', 'single_var']
+    elif 'spect' in tag:
+        dims = ['nAlongDay', 'nAlongNight', 'nAlongGAIMDay', 'nAlongGAIMNight',
+                'single_var']
+
+    # Separate and concatenate into datasets
+    inners = {dim: None for dim in dims}
+
     for path in fnames:
         dataset = xr.load_dataset(path)
+        print(path)
```

Comment:
Don't forget to remove your debug statements.  Alternatively, you can use logging at the 'debug' or 'info' level if you find this information helpful in general.